caringcaribou/modules/uds.py

```diff
@@ -287,11 +287,16 @@ def __uds_discovery_wrapper(args):
     delay = args.delay
     verify = not args.skipverify
     print_results = True
+    padding=args.padding
```

Comment:
The formatting of this block does not adhere to PEP 8, both in terms of spacing and the double newlines at the end